hicommonwealth/commonwealth
# Implement draft save/restore for thread and comment

## Link to Issue
Closes: #3543 

## Description of Changes
- Implements draft save/restore for threads and comments via `draftKey` prop

## Test Plan
- On new discussion page, type text into thread body field, refresh page– confirm previous text is restored
- Navigate between different chains, confirm that the thread body field is restored per-chain
- On view discussion page, type text into new comment field, refresh page- confirm previous text is restored
- Navigate between different view discussion pages, confirm that the comment text is restored per-discussion

## Deployment Plan
N/A

## Other Considerations
- Does not save draft for replies to comments
